<launch>


  <!-- machine file -->
  <include file="$(find pr2_machine)/$(env ROBOT).machine" />

  <!-- ik action -->
  <node pkg="pr2_arm_move_ik" type="arm_ik" name="r_arm_ik" output="screen">
    <param name="joint_trajectory_action" value="r_arm_controller/joint_trajectory_generator" />
    <param name="arm" value="r" />
    <param name="free_angle" value="2" />
    <param name="search_discretization" value="0.01" />
    <param name="ik_timeout" value="5.0" />
  </node>

</launch>

